src/mongoose_client_api/mongoose_client_api_rooms.erl

```diff
@@ -107,8 +107,12 @@ from_json(Req, State = #{was_replied := true}) ->
 from_json(Req, State) ->
     Method = cowboy_req:method(Req),
     {ok, Body, Req2} = cowboy_req:read_body(Req),
-    JSONData = jiffy:decode(Body, [return_maps]),
-    handle_request(Method, JSONData, Req2, State).
+    case mongoose_client_api:json_to_map(Body) of
+        {ok, #{<<"name">> := N, <<"subject">> := S} = JSONData} when is_binary(N), is_binary(S) ->
+          handle_request(Method, JSONData, Req2, State);
```

Comment:
I think this and following 2 lines needs two more spaces of indentation.

Reply:
Done